README.md

```diff
@@ -6,34 +6,47 @@
 
 Marvell patch script to do git patch/apply all open PRs required to build SONIC image
 
-### M0/DNI-ET6448M Platform
+### AC5X-RD Platform
 * SONIC Device: 
-    * armhf-marvell_et6448m_52x-r0
+    * x86_64-marvell_rd98DX35xx-r0
+    	ARCH        | CPU
+    	------------|-----------
+    	x86_64      | Xeon
 
-    ARCH        | CPU
-    ------------|-----------
-    ARMHF       | Armada385
+	* arm64-marvell_rd98DX35xx-r0
+    	ARCH       | CPU
+    	-----------|--------------
+    	ARM64      | ARM-v8.2 A55
 
-* Port: 48x1G+4x10G
+	* arm64-marvell_rd98DX35xx_cn9131-r0
+    	ARCH       | CPU
+    	-----------|-----------
+    	ARM64      | CN913x
 
-```sh
-sonic_M0_master_mvl_patch.sh                - 202006 Jun30 Commit
-```
+
+* Port: 32x1G+16x2.5G+6x25G
 
 ### Falcon Platform
 * SONIC Device: 
     * x86_64-marvell_db98cx8580_16cd-r0 
     * x86_64-marvell_db98cx8580_32cd-r0
-    * arm64-marvell_db98cx8580_16cd-r0 
-    * arm64-marvell_db98cx8580_32cd-r0
+	* x86_64-marvell_db98cx8514_10cc-r0
 
     ARCH        | CPU
     ------------|--------
-    ARM64       | Armada7020
     X86_64      | Xeon
 
-* Port: 12.8T, 6.4T
+* Port: 12.8T, 6.4T, 2T
 
+
```

Comment:
As you had pointed out, we can add TL10 platform here
SONIC Device:
 - x86_64-marvell_dbmvtx9180-r0
  ARCH        | CPU
 ------------|--------
 X86_64        Xeon D15xx

 Port: 6.4T

Reply:
done